test/conftest.py

```diff
@@ -1,11 +1,23 @@
+import os
+
 from pathlib import Path
 
 import pytest
 
 
-@pytest.fixture
-def package_location():
-    return Path(__file__).absolute().parent / 'package'
+@pytest.fixture(params=[True, False], ids=['.', 'absolute-path'])
```

Comment:
I don't love this, but it ensures `shiv .` actually works

Reply:
I definitely like what you are trying to do here! testing `.` !
I'm fine with leaving this as is for now, maybe later we can move this to a `pytest.mark.parametrize` on a specific test or something